src/hb-ot-glyf-table.hh

```diff
@@ -81,26 +80,170 @@ struct glyf
     return_trace (true);
   }
 
-  bool subset (hb_subset_plan_t *plan) const
+  template<typename Iterator,
+	   hb_requires (hb_is_source_of (Iterator, unsigned int))>
+  static bool
+  _add_loca_and_head (hb_subset_plan_t * plan, Iterator padded_offsets)
+  {
+    unsigned int max_offset =
+    + padded_offsets
+    | hb_reduce (hb_max, 0);
+    bool use_short_loca = max_offset <= 131070;
+    unsigned int loca_prime_size = padded_offsets.len () * (use_short_loca ? 2 : 4);
+    char *loca_prime_data = (char *) calloc(1, loca_prime_size);
```

Comment:
So, padded_offsets includes the offset as well?

Instead of doing the multiply yourself and calling calloc with 1, pass two separate numbers to calloc and let it do the mul.

Reply:
done